CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
 # Version changelog
 
+## 0.217.0
+
+CLI:
+ * Added `auth describe` command ([#1244](https://github.com/databricks/cli/pull/1244)).
+ * Fixed message for successful auth describe run ([#1336](https://github.com/databricks/cli/pull/1336)).
+
+Bundles:
+ * Make bundle validation print text output by default ([#1335](https://github.com/databricks/cli/pull/1335)).
+ * Use UserName field to identify if service principal is used ([#1310](https://github.com/databricks/cli/pull/1310)).
+ * Allow unknown properties in the config file for template initialization ([#1315](https://github.com/databricks/cli/pull/1315)).
+ * Remove support for DATABRICKS_BUNDLE_INCLUDES ([#1317](https://github.com/databricks/cli/pull/1317)).
+ * Make `bundle.deployment` optional in the bundle schema ([#1321](https://github.com/databricks/cli/pull/1321)).
+ * Add allow list for resources when bundle `run_as` is set ([#1233](https://github.com/databricks/cli/pull/1233)).
```

Comment:
This change breaks workflows for admins who used `run_as` on their DLT pipelines. Their DABs will now start to fail deployments. Can we highlight that this change is breaking? 

As a side note, we don't have an easy migration path for customers here because the underlying tables also use the "owner" of the DLT pipeline as the identity. Lets wait and see if this impacts more than a few customers.

Reply:
@shreyas-goenka updated

Reply:
Can we also include the context in which it is breaking? That is `bundle deploy` will now start failing if the DAB was deployed by a workspace admin and uses `bundle.run_as` to set the owner of a DLT pipeline.

Reply:
@shreyas-goenka I'd keep Changelog format consistent and don't deviate from ot and rather add the context of the breaking change in the PR description itself
https://github.com/databricks/cli/pull/1233
Could you please describe breaking changes in you PR description?

Reply:
Done, edited the PR description to include the conditions where deployment would break.